app/components/signup-steps/step-one.hbs

```diff
@@ -0,0 +1,28 @@
+<img
+  src='assets/icons/rds-logo.jpg'

```

Comment:
This is already present I think, please use .png or .svg for any image resources or .webp is also good

Reply:
Okk @rohan09-raj will change change in next pr https://github.com/Real-Dev-Squad/website-www/pull/622

Reply:
Done with changes can you please check latest commit